### ISSUE TITLE
INVITES: Remove the 'Join' pop-up for public topics

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1551]
As had been previously discussed, insights from tests show that admins are keen for users to have as few steps as possible before accessing a topic or a group, and they have sometimes expressed concern that the 'Join' pop-up would be off-putting to people: "They don't want to join a group, they just want to come and participate". They see getting people onto a new platform as a major barrier to getting participants, so want it to be as smooth as possible.  

Some participants have also shown hesitation at the 'Join' pop-up. Plus there is [this issue here](https://github.com/citizenos/citizenos-fe/issues/1376). 

We have another issue for improving the pop-ups, but I'm wondering: do we actually need a 'Join' pop-up for public topics and groups? Can we perhaps try removing it, so that following a share link would take them straight to the content, but in order to participate, they then have to join. As soon as they click 'Reply' or 'Add your argument' or whatever, they are asked to Join first (and to log in if they're not already). 

I feel like this would streamline the arrival a lot for public topics/groups, and I can't see that anything important would be lost as a result? @kevincrepin what do you think?

**Comment by BeccaMelhuish** [comment]
Am thinking this is something we might want to test before going live with it though, if we do decide to remove them for public groups, so have added the label as such :)

**Comment by BeccaMelhuish** [comment]
Though, if we do add this text to the pop-up, then I guess this issue is redundant as it will then play quite an important role: https://github.com/citizenos/citizenos-fe/issues/1558

**Comment by kevincrepin** [comment]
I'm OK with this suggestion @BeccaMelhuish . The Join pop-up was there because we also have an actual "Join button" which would then take the user through this little confirmation dialog so they're aware of the rules of conduct.

But there's a difference in sharing/inviting to a topic in that group or the group itself, right? if you're being invited to join a group then shouldn't that popup be the thing you see first? But if you're being invited to a topic, then yeah maybe not? Unless it's a private group?

Actually can't make up my mind on this. But I guess if it lowers the threshold maybe we should just try it and see what the results are. Only in private groups I think you need to join the group before you can see the topics, otherwise a bit pointless to have the private group.

_(edited this after some consideration)_

**Comment by BeccaMelhuish** [comment]
Yeah I agree that for invites / URL links to **public topics** we could remove it and just allow people to access the public topic without joining and without logging in - just like they can if they come to the logged out dashboard. Then if they want to contribute, they will be prompted to login (as is currently the case with the logged out DB). And if they do contribute then they will automatically get 'joined', as per [this issue](https://github.com/citizenos/citizenos-fe/issues/307#issuecomment-2379240791). 

But for public groups we should keep the pop-up (especially as we explain there that their email will be visible to the admins). And for all private topics / groups we keep it too :)  

Will edit the title accordingly, and put this to 'soon'

**Comment by BeccaMelhuish** [comment]
When we do fix this one, it would be important that these issues are fixed before the change: 

- https://github.com/citizenos/citizenos-fe/issues/1636
- https://github.com/citizenos/citizenos-fe/issues/1637

Preferably also this one, especially adding the 'New to Citizen OS?' part:
- https://github.com/citizenos/citizenos-fe/issues/1640

Lower priority, but also related - could be in parallel too:
- https://github.com/citizenos/citizenos-fe/issues/1639

**Comment by BeccaMelhuish** [comment]
Will close this now as the issue is covered by other issues
